plugins/webp-uploads/hooks.php

```diff
@@ -819,3 +820,6 @@ function webp_uploads_render_generator(): void {
 	echo '<meta name="generator" content="webp-uploads ' . esc_attr( WEBP_UPLOADS_VERSION ) . '">' . "\n";
 }
 add_action( 'wp_head', 'webp_uploads_render_generator' );
+
+add_filter( 'wp_content_img_tag', 'webp_uploads_wrap_image_in_picture', 10, 3 );
```

Comment:
Would this make more sense as:

```suggestion
if ( webp_uploads_picture_element_enabled() ) {
	add_filter( 'wp_content_img_tag', 'webp_uploads_wrap_image_in_picture', 10, 3 );
}
```

Reply:
I had this previously, I think I changed it for tests, let me try changed and fix the tests.